src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -13,22 +13,23 @@ class CrudField {
             this.input = this.wrapper.closest("[bp-field-main-input");
             // if no bp-field-main-input has been declared in the field itself,
             // assume it's the first input in that wrapper, whatever it is
-            if (this.input.length == 0) {
+            if (this.input.length === 0) {
                 this.input = $('[bp-field-name="'+ this.name +'"] input, [bp-field-name="'+ this.name +'"] textarea, [bp-field-name="'+ this.name +'"] select').first();
             }
-            this.value = this.input.val();
         }
 
         change(closure) {
-            this.input.change(function(event) {
-                var fieldWrapper = $(this).closest('[bp-field-wrapper=true]');
-                var fieldName = fieldWrapper.attr('bp-field-name');
-                var fieldType = fieldWrapper.attr('bp-field-type');
-                var fieldValue = $(this).val();
+            const fieldChanged = event => {
+                const wrapper = this.input.closest('[bp-field-wrapper=true]');
+                const name = wrapper.attr('bp-field-name');
+                const type = wrapper.attr('bp-field-type');
+                const value = this.input.val();
 
-                // console.log('Changed field ' + fieldName + ' (type '+ fieldType + '), value is now ' + fieldValue);
-                closure(event, fieldValue, fieldName, fieldType);
-            }).change();
+                closure(event, value, name, type);
+            };
+
+            $(this.input).change(fieldChanged);
+            fieldChanged();
```

Comment:
This is a small change, to avoid calling/triggering the `change` event on the input.